### ISSUE TITLE
Error during training on own Dataset

### ISSUE THREAD

**Issue by MohammedAzizi** [pjreddie/darknet#373]
Hi , 

I am trying to overfit 5 images , then during the training phase I got this error after  a certain number of steps:
" ValueError : need at least one array to concatenate " 

Has anyone faced this error before ?

Thanks 

Mohammed

**Comment by sivagnanamn** [comment]
What is the batch size? Are you getting the same error when using batch size 1 (or) < 5?

**Comment by MohammedAzizi** [comment]
thanks for answering,

I fixed this error by introducing an exception , and it seems to work. However i am not able to overfit the 5 images, the loss decreased to nearly 0.05 and no detection was made!

Does it mean that there is no logic in the task i m trying to do ? 

It s  a document image segmentation.

**Comment by sivagnanamn** [comment]
Usually, trying to overfit with very less images is done just used as a sanity check to make sure all the configurations are proper. Since the information you've provided is less, its difficult to guess why you're not getting any detection.

Did you try to test with the same 5 images used for training?
Did you try reducing the threshold used while testing?

**Comment by MohammedAzizi** [comment]
I tested exactly on the 5 training images , but it didn't detect any boxes.

I also tried reducing threshold to 0.06 and I got a lot of wrong detections / grids.
